k8s/packet/k8s-installer/pre_requisite.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+apt-get update && apt-get install -y docker.io apt-transport-https curl
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
+touch /etc/apt/sources.list.d/kubernetes.list
+sh -c 'echo "deb https://apt.kubernetes.io/ kubernetes-xenial main" > /etc/apt/sources.list.d/kubernetes.list'
+apt-get update && apt-get install -y kubelet kubeadm kubectl
+swapoff -a
+
+master() {
+    sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
+    systemctl daemon-reload
+    systemctl restart kubelet
+    kubeadm init --apiserver-advertise-address=$1 --apiserver-cert-extra-sans=10.0.2.15 --pod-network-cidr 10.1.0.0/16
```

Comment:
Would it help to make the cidr configurable ?

Reply:
Yes! So i set the **CIDR** as a variable and store it in vars file